### PR TITLE
Masterbar: Fix compatibility with BeaverBuilder plugin.

### LINKED ISSUE #8594: Masterbar: Fix compatibility issues with BeaverBuilder.
#### Steps to reproduce the issue
With Masterbar enabled edit a layout/page with BeaverBuilder then publish, the Masterbar becomes unclickable.

### PR DESCRIPTION
**Pull request by Pross**
Fixes #8594 

#### Changes proposed in this Pull Request:

* Add filter provided by BeaverBuilder which forces a page refresh on save.

#### Testing instructions:

* Install BeaverBuilder from https://wordpress.org/plugins/beaver-builder-lite-version/
* Make sure WordPress.com toolbar (masterbar) module is active.
* Edit a page with BeaverBuilder, add a module to the page anything you like.
* Click 'Done', then 'Publish'.
* The 'My Sites' link will now be unclickable.

### CODE REVIEW

File: 3rd-party/beaverbuilder.php [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+class BeaverBuilderCompat {
+
+	function __construct() {
+		add_action( 'init', array( $this, 'masterbar_refresh' ) );
+	}
+
+	/**
+	 * If masterbar module is active force BeaverBuilder to refresh when publishing a layout.
+	 */
+	function masterbar_refresh() {
```
Comment (jeherve) [inline 3rd-party/beaverbuilder.php]: Do you think you could name this a bit differently, since we are refreshing the builder, and not the masterbar?

Comment (Pross) [reply 3rd-party/beaverbuilder.php]: Not a problem.

**Comment by Pross** [comment]
Not sure why travis failed, just seemed to timeout.

**Comment by jeherve** [comment]
Thanks. I just relaunched the Travis tests to get that solved.

**Pull request merged**